### DISCUSSION TITLE
NodeJS example

### DISCUSSION THREAD

**Discussion by nshiab** [duckdb/duckdb-wasm#1417] [Ideas]
Hi!

First, thanks for this fantastic piece of technology and for all of your work. 🙏 

Would it be possible to add a NodeJS example to your readme?

Thanks!

**Comment by Mause** [comment]
Have you had a look at the examples folder? https://github.com/duckdb/duckdb-wasm/tree/main/examples

**Reply by nshiab** [reply]
Ah! Perfect, thanks. Sorry to have bothered you... I should have checked. 🤦
